app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_user_posts_count
+
+  def update_user_posts_count
+    author.update_posts_count
+  end
+
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I really appreciate what you have done 😄, It would be better if you could make the `Post` model method that updates the posts counter for a user as follows:
```
# app/models/post.rb
...

def update_post_counter
  author.increment!(:posts_counter)
end
...
```

Reply:
Could you please explain what's that ?